test/functional/sqlsrv/MsSetup.inc

```diff
@@ -44,4 +44,11 @@ if (isset($_ENV['MSSQL_SERVER']) || isset($_ENV['MSSQL_USER']) || isset($_ENV['M
 $keystore = "none";         // key store provider, acceptable values are none, win, ksp, akv
 $dataEncrypted = false;     // whether data is to be encrypted
```

Comment:
So here, possible values for $keystore can be `akv_key`, `akv_pwd` or whatever you have defined in MsHelper.inc / MsCommon-mid-refactor, etc. These two values will identify the test branches in test lab